File: Algoritmo(C)/Alg_S6_8.c
#include <stdio.h>
int main(){
    //variables
    int n;
    //input
    printf("Digite um numero: ");
    scanf("%d",&n);
    //process + output
    if (n % 2 == 0){
        if (n > 0){
            printf("O numero %d e PAR e POSITIVO",n);
        }else{
            printf("O numero %d e PAR e NEGATIVO",n);
        }
    }else{
        if (n > 0){
            printf("O numero %d e IMPAR e POSITIVO",n);
        }else{
            printf("O numero %d e IMPAR e NEGATIVO",n);
        }
    }
}

File: Algoritmo(Python)/Alg_S6_8.py
#input
num = int(input("Digite um número: "))
#process
if(num % 2 == 0):
    print("Este número é PAR")
    if(num > 0):
        print("Este número é POSITIVO")
    else:
        print("Este número é NEGATIVO")
else:
    print("Este número é IMPAR")
    if(num > 0):
        print("Este número é POSITIVO")
    else:
        print("Este número é NEGATIVO")